modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -27,6 +27,22 @@ You can configure Anypoint Service Mesh using the command line interface (CLI) o
 * xref::configure-using-CLI.adoc[Using CLI]
 * xref::configure-using-CRD.adoc[Using CRD]
 
+== Configuring Response Headers for Anypoint Service Mesh
+
+Services protected by Anypoint Service Mesh return extra headers. These headers are added by Envoy and the gRPC adapter.
+
+
+You can remove these headers to avoid returning them to the originating client.
```

Comment:
Should we say something about why or under what conditions the audience wants or needs to do this?

Reply:
It is configured if the users don't want extra headed to be returned to the originating API. I think that comes across clearly. Done! Thanks!